src/test/java/hudson/plugins/performance/parsers/JMeterCsvParserTest.java

```diff
@@ -43,11 +43,11 @@ public void canParseCsvFileWhenSkipFirstLineIsNotSpecifiedAndFirstLineHasHeader(
     public void testDateDateFormats() throws Exception {
         final JMeterCsvParser parser = new JMeterCsvParser(NO_GLOB, PerformanceReportTest.DEFAULT_PERCENTILES);
         PerformanceReport performanceReport = parseAndVerifyResult(parser, reportFile);
-        assertEquals(41.9, performanceReport.getMaxKb(), 0.01);
+        assertEquals(19.25, performanceReport.getMaxKb(), 0.01);
```

Comment:
if we are calling the same method, why would this value have changed?

Reply:
I changed this method from "return the sum of all byte values" to "return the maximum of all byte values". The largest test in the csv unit test resource is 19.25KB, so this test should assert on that instead